scylla/src/transport/partitioner.rs

```diff
@@ -10,6 +10,23 @@ pub(crate) enum PartitionerName {
     CDC,
 }
 
+impl PartitionerName {
+    pub(crate) fn from_str(name: &str) -> Option<Self> {
+        if name.ends_with("CDCPartitioner") {
+            Some(PartitionerName::CDC)
+        } else {
+            None
```

Comment:
I'm aware that the only call site of this code uses `unwrap_or_default`, but I think we shouldn't rely on that and just copy the original logic by returning PartitionerName::Murmur3 here. Later, if we ever add any more partitioners, we could use None as actually signaling that we failed to find a matching partitioner

Reply:
ping, what do you think about this suggestion?

Reply:
IMO, the original logic is better preserved by returning None, but I don't have strong opinion on it, so I'm ok to modify it. However, should I keep the `Option` in the signature return then?

Reply:
How about that - let's keep the Option in the signature, but add an explicit branch which checks if `name.ends_with("Murmur3Partitioner")`. That way, we're explicit that we're capable of detecting Murmur3Partitioner correctly, and really return None if we didn't manage to detect the partitioner at all.

Reply:
Let's do this.

Reply:
done